polkadot/node/core/pvf/src/testing.rs

```diff
@@ -59,21 +59,21 @@ pub fn validate_candidate(
 ///
 /// NOTE: This should only be called in dev code (tests, benchmarks) as it relies on the relative
 /// paths of the built workers.
-pub fn build_workers_and_get_paths(is_bench: bool) -> (PathBuf, PathBuf) {
+pub fn build_workers_and_get_paths(release_build: bool) -> (PathBuf, PathBuf) {
```

Comment:
I believe we can remove this now, right?

And as always, thank you so much for the contribution!